common/src/test/scala/uk/ac/wellcome/platform/archive/common/bagit/models/BagInfoTest.scala

```diff
@@ -1,148 +1,99 @@
-      )
-    )
-  }
-
-  it(
-    "returns a left of invalid bag info error if there is no external identifier in bag-info.txt"
-  ) {
-    val bagInfoString =
-      s"""|Source-Organization: $randomSourceOrganisation
-          |Payload-Oxum: ${createPayloadOxum.payloadBytes}.${createPayloadOxum.numberOfPayloadFiles}
-          |Bagging-Date: $randomLocalDate""".stripMargin
-
-    BagInfo.parseBagInfo(t, IOUtils.toInputStream(bagInfoString, "UTF-8")) shouldBe Left(
-      InvalidBagInfo(t, List("External-Identifier"))
-    )
-  }
-
-  it(
-    "returns a left of invalid bag info error if there is no payload-oxum in bag-info.txt"
-  ) {
-    val bagInfoString =
-      s"""|External-Identifier: $createExternalIdentifier
-          |Source-Organization: $randomSourceOrganisation
-          |Bagging-Date: $randomLocalDate""".stripMargin
-
-    BagInfo.parseBagInfo(t, IOUtils.toInputStream(bagInfoString, "UTF-8")) shouldBe Left(
-      InvalidBagInfo(t, List("Payload-Oxum"))
-    )
-  }
-
-  it(
-    "returns a left of invalid bag info error if the payload-oxum is invalid in bag-info.txt"
-  ) {
-    val bagInfoString =
-      s"""|External-Identifier: $createExternalIdentifier
-          |Source-Organization: $randomSourceOrganisation
-          |Payload-Oxum: sgadfjag
-          |Bagging-Date: $randomLocalDate""".stripMargin
-
-    BagInfo.parseBagInfo(t, IOUtils.toInputStream(bagInfoString, "UTF-8")) shouldBe Left(
-      InvalidBagInfo(t, List("Payload-Oxum"))
-    )
-  }
-
-  it(
-    "returns a left of invalid bag info error if there is no bagging date in bag-info.txt"
-  ) {
-    val bagInfoString =
-      s"""|External-Identifier: $createExternalIdentifier
-          |Source-Organization: $randomSourceOrganisation
-          |Payload-Oxum: ${createPayloadOxum.payloadBytes}.${createPayloadOxum.numberOfPayloadFiles}""".stripMargin
-
-    BagInfo.parseBagInfo(t, IOUtils.toInputStream(bagInfoString, "UTF-8")) shouldBe Left(
-      InvalidBagInfo(t, List("Bagging-Date"))
-    )
-  }
-
-  it(
-    "returns a left of invalid bag info error if the bagging date is invalid in bag-info.txt"
-  ) {
-    val bagInfoString =
-      s"""|External-Identifier: $createExternalIdentifier
-          |Source-Organization: $randomSourceOrganisation
-          |Payload-Oxum: ${createPayloadOxum.payloadBytes}.${createPayloadOxum.numberOfPayloadFiles}
-          |Bagging-Date: sdfkjghl""".stripMargin
-
-    BagInfo.parseBagInfo(t, IOUtils.toInputStream(bagInfoString, "UTF-8")) shouldBe Left(
-      InvalidBagInfo(t, List("Bagging-Date"))
-    )
-  }
-
-  it("returns a left of invalid bag info error if bag-info.txt is empty") {
-    val bagInfoString = ""
-
-    BagInfo.parseBagInfo(t, IOUtils.toInputStream(bagInfoString, "UTF-8")) shouldBe Left(
-      InvalidBagInfo(
-        t,
-        List("External-Identifier", "Payload-Oxum", "Bagging-Date")
-      )
-    )
-  }
-
-}
+//package uk.ac.wellcome.platform.archive.common.bagit.models
```

Comment:
Think you might have meant to remove this.

Reply:
🤦‍♀ 

Oops, will fix, thanks.